Makefile

```diff
@@ -29,7 +29,7 @@ ifeq (${LWIP_G3_BUILD}, yes)
 include g3/Makefile
 endif
 
-CFLAGS += -Wundef -Iinclude -Ilib-lwip/src/include -I"$(LWIPOPTS_DIR)"
+CFLAGS += -Wundef -Wno-char-subscripts -Iinclude -Ilib-lwip/src/include -I"$(LWIPOPTS_DIR)"
```

Comment:
Big nope. You disable this warning globally, also for our port part.

Reply:
Moved this to `LOCAL_CFLAGS`